products/batch_exports/backend/temporal/destinations/snowflake_batch_export.py

```diff
@@ -134,6 +137,12 @@ class SnowflakeRetryableConnectionError(Exception):
     pass
 
 
+class SnowflakeWarehouseSuspendedError(Exception):
+    """Raised when a Warehouse is suspended."""
+
+    pass
```

Comment:
nit: Could handle exception message here.

Reply:
(we'd have to pass in `max_attempts` for example, which feels a bit messy)